solutions/dashboard/charts/dashboard-project/values.yaml

```diff
@@ -0,0 +1,154 @@
+tools:
+  namespace: ""
+  dashboard_secret_name: ""
+  dashboard_configmap_name: ""
+  create:
+    portscanner: true
+    codacy: true
+    github: true
+    jenkins: true
+
+dashboard_configmap:
+  elasticsearch_http_service: "dashboard-elasticsearch-es-http.elastic.svc"
+  elasticsearch_port: "9200"
+  mongodb_connection_url: "dashboard-mongodb.dashboard.svc.cluster.local:27017"
+  github_api_baseurl: "https://api.github.com"
+  jenkins_server_url: ""
+
+codacy:
+  name: ""
+  schedule: "0 0 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  image: cortx-docker.colo.seagate.com/dashboard/codacy:1.0.0-latest
+  imagePullPolicy: Always
+  resources:
+    limits:
+      memory: "768Mi"
+      cpu: "5000m"
+    requests:
+      memory: "256Mi"
+      cpu: "300m"
+  restartPolicy: OnFailure
+
+github:
+  name: ""
+  schedule: "0 0 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  image: cortx-docker.colo.seagate.com/dashboard/github:1.0.0-latest
+  imagePullPolicy: Always
+  resources:
+    limits:
+      memory: "2048Mi"
+      cpu: "5000m"
+    requests:
+      memory: "256Mi"
+      cpu: "200m"
+  restartPolicy: OnFailure
+
+jenkins:
+  name: ""
+  schedule: "0 0 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  image: cortx-docker.colo.seagate.com/dashboard/jenkins:1.0.0-latest
+  imagePullPolicy: Always
+  resources:
+    limits:
+      memory: "2048Mi"
+      cpu: "5000m"
+    requests:
+      memory: "256Mi"
+      cpu: "200m"
+  restartPolicy: OnFailure
+
+logstash_configmap:
+  name: ""
+
+logstash:
+  name: ""
+  schedule: "0 */5 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  image: docker.io/sanket02092001/customlog:1.0.0
+  imagePullPolicy: Always
+  resources:
+    limits:
+      memory: "2048Mi"
+      cpu: "5000m"
+    requests:
+      memory: "256Mi"
+      cpu: "200m"
+  restartPolicy: OnFailure
+
+mongodb:
+  name: ""
+  replicas: 1
+  image: mongo:6.0.2
```

Comment:
Let's keep this image also in the Harbor repo to avoid external dependency.

Reply:
Ok, Sure

Reply:
@Sanket292001 Uploaded image in local Harbor at `cortex-docker.colo.seagate.com/dashboard/mongo:6.0.2` . Please use this image and validate the deployment.

Reply:
Tested the MongoDB deployment with `cortx-docker.colo.seagate.com/dashboard/mongo:6.0.2` image. It is working properly. 
I will push the modified changes.

Reply:
Hi @shailesh-vaidya, Pushed the modified code. Please check